.github/actions/comment-pr-with-links/action.yml

```diff
@@ -0,0 +1,102 @@
+name: 'Comments PR with useful links'
+description: 'Add a comment to help reviewer on a PR'
+
+inputs:
+  site-url:
+    description: 'The url of preview to check the PR update'
+    required: true
+  component-name:
+    description: 'The name of the component to build. If set, the build-preview-command input is ignored'
+    required: false
+    default: 'bonita'
```

Comment:
⚠️ I would make the input required without default. The action is going to be used only internally, and we will always pass the component

Reply:
Let keep the default value.